app/index.html

```diff
@@ -3,28 +3,54 @@
 <head>
     <meta charset="UTF-8">
     <title>Image Stitcher by Markus Zeller</title>
```

Comment:
Please add your name, if you like.

Reply:
No. But we can simply replace it by version and introduce a constant for version?

Reply:
I'd like to keep version number out here, because it needs to be maintained at git release tag, const and HTML.

Reply:
Okay. So we take out the names as well.

Reply:
Better move to an author meta tag like `<meta name="author" content="John Doe, Buck Rogers">`.